CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # CHANGELOG
 
-## Next Release
+## v0.9.0 (2023-05-17)
 
 - Fix a bug where URLs were not being extracted correctly, potentially causing false matches when matching by URL
+- New match rules for matching by query parameters, by path or by custom user-defined rules
```

Comment:
nit: prefer to list features then fixes, reads better since it's a minor bump, the expectation is it introduces new features so to read fix first seems backwards